ios/flutter_vlc_player.podspec

```diff
@@ -1,24 +1,25 @@
 #
-# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
```

Comment:
Since we are touching this file. What do you think of changing it a little ?

```
  s.version          = '2.0.0' // > Equal to pubspec
  s.summary          = 'A new Flutter project.'
  s.description      = <<-DESC	  s.description
A VLC-powered alternative to Flutter's video_player. Supports multiple players on one screen.
//....
s.homepage         = 'https://github.com/solid-software/flutter_vlc_player'
```

Reply:
If we follow google standard https://github.com/flutter/plugins/blob/master/packages/battery/ios/battery.podspec

We could use something like 

```
A Flutter plugin to access various information about the battery of the device the app is running on.
Downloaded by pub (not CocoaPods).
```

Reply:
resolved